simd/src/Kokkos_SIMD_Common.hpp

```diff
@@ -154,6 +154,14 @@ template <class T>
   return const_where_expression(mask, value);
 }
 
+// fallback simd multiplication using generator constructor
+
+template <class T, class Abi>
+[[nodiscard]] KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION simd<T, Abi> operator*(
```

Comment:
Why is this needed?  What about other arithmetic operators?

Reply:
It is needed because AVX2 does not support multiplication of a vector of 64-bit integers. We could add the other operators for consistency if desired, there are just no backends yet that would use them.

Reply:
@PhilMiller do you mean a comment in the C++ source where the fallback is defined or in the merge request message?

Reply:
I meant a source code comment